02_configure_host.sh

```diff
@@ -29,9 +29,17 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
 
 # Allow local non-root-user access to libvirt
 # Restart libvirtd service to get the new group membership loaded
+OS=$(uname -a)
 if ! id $USER | grep -q libvirt; then
-  sudo usermod -a -G "libvirt" $USER
-  sudo systemctl restart libvirtd
+  if [[$OS == *Ubuntu* ]]; then
+    originalGRP=$(id -g -n)
+    # A trick to add 'libvirt' group to user without reloginrelogin 
+    newgrp libvirt
+    newgrp $originalGRP
```

Comment:
Some actions need to take place between the two newgrp commands. 
Or remove the second one as it only does restoring.